### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by UltimaBGD**
In regards to issue #347, these are the issue templates for bug reports and enhancements. They are built with the default GitHub templates and the ones used by Kubernetes in mind.

These can likely be improved and iterated on further. If there are any other things to add that would be useful, please let me know.

### CODE REVIEW

**Comment by UltimaBGD** [comment]
Interestingly, despite my PR only touching a new folder that should only be read by Github and not impact the rest of the project at all, it appears to have failed some tests on Travis and Travis alone. Looking at the logs, it appears the tests for text-parser.js timed out on the windows and mac builds. I believe this is the same instance of bugs as #439, and now we know it is not just locally.

**Comment by cindyledev** [comment]
It passed both tests now

**Comment by UltimaBGD** [comment]
Right. It’s intermittent timeout issues that are talked about in the issue I linked.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+name: Bug Report
+about: Create a report to help us improve Telescope
+title: ''
+labels: kind/bug
```
Comment (humphd) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Should this be `type: bug`? to match what we have in our labels.  I'm  not sure.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+assignees: ''
```
Comment (humphd) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Do we need to include this if it's just going to be left blank?

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+name: Bug Report
+about: Create a report to help us improve Telescope
+title: ''
```
Comment (humphd) [inline .github/ISSUE_TEMPLATE/bug-report.md]: If blank, is it useful?

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+**What happened**:
+
+**What should have happened**:
+
+**How to reproduce it (as precise as possible)**:
+
+**Anything else we need to know?**:
+
+**Environment**:
+- OS:
```
Comment (humphd) [inline .github/ISSUE_TEMPLATE/bug-report.md]: What about adding Browser to this list?

File: .github/ISSUE_TEMPLATE/enhancement.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+---
+name: Enhancement
+about: Suggest an idea for Telescope
+title: ''
```
Comment (humphd) [inline .github/ISSUE_TEMPLATE/enhancement.md]: Let's remove this, since it's blank.

File: .github/ISSUE_TEMPLATE/enhancement.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+labels: kind/feature
```
Comment (humphd) [inline .github/ISSUE_TEMPLATE/enhancement.md]: Let's fix this to match our label: `type: enhancement`

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (UltimaBGD) [reply .github/ISSUE_TEMPLATE/bug-report.md]: I tested type: bug, and I don't think it likes that formatting due to labels already using the semicolon.

Or I may be misunderstanding and you mean replacing the entire thing with type: bug?

Comment (UltimaBGD) [reply .github/ISSUE_TEMPLATE/bug-report.md]: I got it, it needed to be enclosed in quotes in order to not be read as an additional specifier.

**Pull request merged**